client/js/is-sum-function-test.js

```diff
@@ -0,0 +1,20 @@
+const { isValidFormat: isValidFormat,
+	      isSumFunction: isSumFunction,
```

Comment:
combo of tabs and spaces is messing up your indent

![image](https://cloud.githubusercontent.com/assets/17621133/25734358/c07d2422-3116-11e7-9d51-03745ed8e323.png)